BiCo/src/app/components/negocios/mis-negocios/mis-negocios.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { MisNegociosComponent } from './mis-negocios.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ISPP-2021/BiCo-Frontend/pullRequest?prid=7237503)
```suggestion
import { MisNegociosComponent } from "./mis-negocios.component";
```